schema.h

```diff
@@ -208,13 +209,14 @@ struct ResponseCardEntry {
 
 // Universal response structure, combining user info & cards payload.
 struct ResponseFeed {
-  uint64_t ms;                           // Server timestamp, milliseconds from epoch.
-  ResponseUserEntry user;                // User information.
-  std::vector<ResponseCardEntry> cards;  // Cards feed.
+  uint64_t ms;                                  // Server timestamp, milliseconds from epoch.
+  ResponseUserEntry user;                       // User information.
+  std::vector<ResponseCardEntry> hot_cards;     // "Hot" cards feed.
```

Comment:
`feed_hot`, `feed_recent`?
Or perhaps `feeds` -> `map<string, vector<ResponseCardEntry>>`? We gotta add geography and "nearby" some time in the future.